src/vendors/smartstake/validators_dump.js

```diff
@@ -0,0 +1,6932 @@
+      commission: 5.0,
+      risky: null,
+      stakeWeight: 0.014,
+      currentPerSigned: 100.0,
+      currentSigned: 120,
+      currentMissed: 0,
+      currentLatestSigned: 7509977,
+      currentLatestProposed: 7203787,
+      shortPerSigned: 100.0,
+      shortSigned: 4928,
+      shortMissed: 0,
+      shortLatestSigned: 7509977,
+      shortLatestProposed: 7203787,
+      dailyPerSigned: 100.0,
+      dailySigned: 5011,
+      dailyMissed: 0,
+      dailyLatestSigned: 7509977,
+      dailyLatestProposed: 7203787,
+      midPerSigned: 52.78,
+      midSigned: 9204,
+      midMissed: 10217,
+      midLatestSigned: 7509977,
+      midLatestProposed: 7203787,
+      longPerSigned: 68.32,
+      longSigned: 53023,
+      longMissed: 26653,
+      longLatestSigned: 7509977,
+      longLatestProposed: 7203787
+    }
+  ],
+  lastUpdated: 330,
+  oasisChainId: 'oasis-1',
+  coinStat: {
+    totalStake: 5781081600.0,
+    stakingPools: 192,
+    currentRewardRate: 15.0,
+    ssStake: null,
+    totalSupply: '10000000000',
+    circulatingSupply: '3490000000',
+    symbol: 'OASIS',
+    lastUpdated: '2022-03-04T20:18:38',
+    btcPrice: 5.92e-6,
+    usdPrice: 0.23614,
+    ssBlocks: null,
+    ssRewards: null,
+    recentBlock: 7510678,
+    blocks_8hrs: null,
+    blocks_24hrs: null,
+    blocks_7days: null,
+    blocks_1hr: null,
+    withdrawSyncTillBlock: null,
+    uniqueDelegates: 165,
+    coinId: 3,
+    totalRewardWithdrawals: null,
+    activeAccounts: 22251,
+    totalPoolStake: null,
+    totalSoloStake: null,
+    percentStaked: 57.8,
+    usdStaked: null,
+    usdMcap: 822796000.0,
+    priceChangeUsd: -0.01,
+    rewardsBalance: null,
+    blocks_21days: null,
+    actualRewardRate: null,
+    blockRate: null,
+    medianRawStake: null,
+    isSynced: '1',
+    stakeSupply: 1388320384.0,
+    electedPools: null,
+    totalWeightedStake: null,
+    otherArr: null,
+    smartStakeFee: 19.5,
+    medianFee: 20.0,
+    overallApr: null,
+    valForNetworkHalt: 9,
+    epochTimestamp: 1608432525,
+    signRate: null,
+    totalTransactions: null,
+    epochLastBlock: null,
+    nextEpochTime: null,
+    currentEpoch: 12506,
+    assetCount: null,
+    uniqueDelegations: null,
+    tvl: null,
+    status: 'ACTIVE',
+    eco: null,
+    geckoId: 'oasis-network',
+    mcapRank: 101,
+    fdvMcap: 2357580000.0,
+    maxSupply: 10000000000.0,
+    tokenSymbol: 'ROSE',
+    gini: 0.529691,
+    tvlInd: 'false',
+    avgVotingPower: 0.900901,
+    valBelowAvgPower: 71,
+    nakamotoIndex: 20,
+    sync: 'true',
+    weightedMedianCommission: 20.0,
+    analyticsWebsite: 'https://oasis.smartstake.io',
+    consensusValidators: 111
+  },
+  oasisBlockHeight: '7510677',
+  notification: null,
+  statusSummary: { InConsensus: '111', NotInConsensus: '81', AllVal: 192 }
+}
+
+export const validators = validatorsJson.data.map((v, ix) => {
+  return {
+    name: v.entityName ?? undefined,
+    address: v.idAddress,
+    escrow: v.total * 10 ** 9,
```

Comment:
I could also add `current_rate: v.commission != null ? v.commission / 100 : undefined` but that is a bit more sensitive if it is out of date

Reply:
In combination with a general notification like "Showing validator list as of <DATE-AND-TIME>" I think it would be fine to show it.

Reply:
Added
![localhost_3000_account_oasis1qp25k3hv6ysrm6wntjnt6x0j5du0ud4qqchplrf2_stake (1)](https://user-images.githubusercontent.com/3758846/157794550-2fd1b23d-a476-4cb9-b1dd-e236ff38180a.png)